git_machete/github.py

```diff
@@ -180,8 +180,10 @@ def __fire_github_api_request(method: str, path: str, token: Optional[str], requ
                                  'Visit `https://github.com/settings/tokens` to generate a new one.')
         elif err.code == http.HTTPStatus.NOT_FOUND:
```

Comment:
Actually... is there any reason 404 is not included in the case above (401 & 403)?

Reply:
That would be much simpler and would avoid repetitive messages

Reply:
@PawelLipski I don't know what was the initial reason the separate them but i understand why, even though both are related to token, there is a slight difference between them (present but not valid token vs token not present but required). I would say that it's worth to split them despite repetitive messages, it's much easier for the user to understand why it fails and what he needs to do to fix it, wdyt?

Reply:
Okay, makes sense, just keep the messages consistent (unless ofc there's a good reason to keep it different)